src/components/course/data/hooks.jsx

```diff
@@ -694,16 +698,15 @@ export const useUserSubsidyApplicableToCourse = ({
 
     let applicableUserSubsidy;
 
-    // if course can be redeemed with EMET system, return
-    // the redeemable subsidy access policy.
-    if (isPolicyRedemptionEnabled && redeemableSubsidyAccessPolicy) {
+    // if course can be redeemed with a subsidy access policy, return it as the subsidyType.
+    if (isPolicyRedemptionEnabled) {
       applicableUserSubsidy = {
         discountType: 'percentage',
         discountValue: 100,
         subsidyType: LEARNER_CREDIT_SUBSIDY_TYPE,
-        perLearnerEnrollmentLimit: redeemableSubsidyAccessPolicy.perLearnerEnrollmentLimit,
-        perLearnerSpendLimit: redeemableSubsidyAccessPolicy.perLearnerSpendLimit,
-        policyRedemptionUrl: redeemableSubsidyAccessPolicy.policyRedemptionUrl,
+        perLearnerEnrollmentLimit: redeemableSubsidyAccessPolicy?.perLearnerEnrollmentLimit,
+        perLearnerSpendLimit: redeemableSubsidyAccessPolicy?.perLearnerSpendLimit,
+        policyRedemptionUrl: redeemableSubsidyAccessPolicy?.policyRedemptionUrl,
```

Comment:
[context] The enterprise-access `can-redeem` API returns `can_redeem: false` when a learner has already redeemed a course. This means the course page thinks the learner no longer has any subsidy available to spend. Now, `isPolicyRedemptionEnabled` is true when `can_redeem: false and has_successful_redemption: true`, but that means there is no `redeemableSubsidyAccessPolicy` so that needs to be handled in this object.

Reply:
This might better serve us long term as a comment?

Reply:
Couldn't hurt! Will move it.